proxy/envoy/testdata/envoy-v0.json.golden

```diff
@@ -1,7 +1,53 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:1081",
+      "address": "tcp://0.0.0.0:80",
```

Comment:
We'd have to change the way we were checking against our golden files. Right now we do a simple diff between the golden files and the generated files to check for issues.